Documentation/client.md

```diff
@@ -13,7 +13,11 @@ If you don't yet have a Rook cluster running, refer to our [Quickstart Guides](R
 2. Map the block volume and format it and mount it
 
     ```bash
-    sudo -E rook block map --name test --format --mount /tmp/rook-volume
+    # If running in the rook-client container, no need to run privileged
+    rook block map --name test --format --mount /tmp/rook-volume
```

Comment:
at first read of this diff, i was concerned about removing the sudo because standalone users would run into a permissions issue right away.  however, after looking at the rendered markdown at https://github.com/travisn/rook/blob/alpine-client/Documentation/client.md, I think the comments are visible enough to easily enough help the user run the right command.